cmd/skopeo/main.go

```diff
@@ -102,6 +102,8 @@ func main() {
 		return
 	}
 	rootCmd, _ := createApp()
+	// Disable the implicit `completion` command in cobra.
```

Comment:
(I’m not sure this comment helps much. Pointing at the existing `completions/bash/skopeo` script would at least document the rationale.)

Reply:
Better?

Reply:
Sure. Thanks!